.github/workflows/security-dependency-review.yml

```
@@ -0,0 +1,25 @@
+jobs:
+  dependency-review:
+    name: Dependency Review
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
+
+    - name: Dependency Review
+      uses: actions/dependency-review-action@2031cfc080254a8a887f58cffee85186f0e49e48 # v4.9.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)